src/gs1/commands/verify-integrity.command.ts

```
@@ -0,0 +1,75 @@
+        console.log('\n⚠️ Integrity check failed for one or more files! ⚠️');
+        console.log('This indicates that files may have been tampered with or corrupted.');
+      } else {
+        console.log('\n✓ All integrity checks passed successfully.');
+      }
+      
+    } catch (error: unknown) {
+      const errorMessage = error instanceof Error ? error.message : String(error);
+      this.logger.error(`Failed to verify integrity: ${errorMessage}`);
+    }
+  }
+} 
```

ESLint

[error] Replace `·` with `⏎`

(prettier/prettier)